website/docs/optimizations/java/results.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Results of the optimization steps"
+sidebar_position: 6
+---
+
+For the different labs, we looked at one optimization at a time: we started with the initial container image without optimization, used jlink and jdeps for a custom runtime, introduced jib, and finally used GraalVM. The following table shows the different versions with the image size and the start time of the application (tested using Amazon EKS).
```

Comment:
In previous labs, we looked at one optimization at a time. We started with the initial container image without optimization, then used 'jlink' and 'jdeps' for a custom runtime, introduced 'jib', and finally used GraalVM. The following table shows the different versions with the image size and the start time of the application (tested using Amazon EKS).